src/birthplace.js

```diff
@@ -128,10 +128,10 @@ function parseMalaysia(code) {
 }
 
 function parseForeign(code) {
-    return Object.assign(
-        { state: null },
-        countryCodePairs[code]
-    );
+    return {
```

Comment:
`Object.assign` is preferred for native support on a wide amount of browsers. Spread operator `...` would get transpiled to `_objectSpread` (see above) for very old browsers  that are too old to support and adds to the bundle size.